README.md

```diff
@@ -73,7 +73,7 @@ func main() {
     for {
     	// Use iterator.Next to retrieve next object until we meet IteratorDone.
     	o, err := it.Next()
-    	if errors.Is(err, types.IteraoorDone) {
+    	if errors.Is(err, types.IterateDone) {
```

Comment:
Can you fix the errors in the comments and Chinese document together?

Reply:
I've updated the comment line in question along with the Chinese read-me. In Chinese one, I've just updated the type, though.